hw_3/hw_3.2.js

```diff
@@ -6,25 +6,23 @@ Task2*. Решить квадратные уравнения. Переменны
 */
 
 // quadraticEquation1 -> x2 - 6x + 9 = 0; 
-let x1;
-let x2;
 
-let a1 = 1;
-let b1 = -6;
-let c1 = 9;
+const a1 = 1;
+const b1 = -6;
+const c1 = 9;
 
-let discriminant1 = b1 ** 2 - 4 * a1 * c1;
+const discriminant1 = b1 ** 2 - 4 * a1 * c1;
```

Comment:
Or you could use `let`, but reasign them and reuse 
`const discriminant = b ** 2 - 4 * a * c;` for second equation